.travis.yml

```diff
@@ -1,31 +1,23 @@
 dist: xenial
 language: go
-
 go:
-  - 1.12.x
-
+- 1.12.x
 sudo: required
-
 services:
-  - docker
-
+- docker
 script:
-  - export BRANCH=$(if [ "$TRAVIS_PULL_REQUEST" == "false" ]; then echo $TRAVIS_BRANCH; else echo $TRAVIS_PULL_REQUEST_BRANCH; fi)
-  - echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, PR=$TRAVIS_PULL_REQUEST, BRANCH=$BRANCH"
-  - hack/ci-check.sh
```

Comment:
Hmm, looks like the `hack/ci-check.sh` script step was dropped, so we're not getting a `make ci` run before moving on to building/pushing container images -- was that intentional? I'm thinking we probably want to retain it, so we get all units run.